BonusCalcApi/V1/Gateways/OperativeGateway.cs

```diff
@@ -24,5 +26,19 @@ public async Task<Operative> GetOperativeAsync(string operativeId)
                 .ThenInclude(s => s.PayBands)
                 .SingleOrDefaultAsync(o => o.Id == operativeId);
         }
+
+        public async Task<IEnumerable<Operative>> GetOperativesAsync(string query, int? page, int? size)
+        {
+            int pageNumber = Math.Max(Math.Min((page ?? 1), 100), 1);
```

Comment:
Would it be clearer to use Math.Clamp here?

Reply:
Yes, didn't know it existed 👍🏻